### PR TITLE
8 describe dataset

### LINKED ISSUE #8: Describe Dataset
# Feature
Generate Descriptive Statistics

# Description
As a data scientist, I want to generate descriptive statistics for all numerical features in the dataset, So that I can understand the distribution and variability of the data.

# Acceptance Criteria:
- [x] The program should be named describe.[extension].
- [x] The program should accept a dataset file as input.
- [x] The output should include count, mean, standard deviation, min, 25%, 50%, 75%, and max for each numerical feature.

# Output example:

Image

### PR DESCRIPTION
**Pull request by farinaleo**
add the describe required function.
See issue for usage.

### CODE REVIEW

File: srcs/descibe/tools.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import pandas as pd
+
+
```
Comment (kazourak) [inline srcs/descibe/tools.py]: this function only call read_csv pandas function, why ?

**Review by kazourak — CHANGES_REQUESTED** [review]
the directory name is "descibe" instead of "describe".

File: srcs/descibe/tools.py (hunk 1 above)
Comment (farinaleo) [reply srcs/descibe/tools.py]: more simple if we decide to normalise data.

**Comment by farinaleo** [comment]
> the directory name is "descibe" instead of "describe".
Thanks, problem fixed

**Review by kazourak — APPROVED** [review]
thanks, great.

**Pull request merged**